project/Dependencies.scala

```diff
@@ -31,6 +31,15 @@ object Dependencies {
     )
   }
 
+  val pekko = Def.setting[Seq[ModuleID]] {
+    val ver = "1.0.1"
+    Seq(
```

Comment:
```suggestion

    Seq(
```

Reply:
Addressed in 87d3301898f6bbf86f7167a2c141cc2bbd9e8370